docs/rules/id-length.md

```diff
@@ -89,8 +91,7 @@ class x { }
 class Foo { x() {} }
 function foo(...x) { }
 var { x } = {};
-var { x: a} = {};
-var { a: [x]} = {};
```

Comment:
Is this a regression in this PR? I see [that this is going to be worked on in a separate PR](https://github.com/eslint/eslint/pull/12881#issuecomment-584330660). Do we have an issue outlining our plan for this (just in case someone else needs to pick this work up)?

Reply:
> Is this a regression in this PR?

Yes. The case will not be warned after this PR merged because the actual version warns about `a` rather than `x`. 

* [Online Demo](https://eslint.org/demo#eyJ0ZXh0IjoiLyplc2xpbnQgaWQtbGVuZ3RoOiBbXCJlcnJvclwiLCB7IFwibWluXCI6IDQgfV0qL1xuLyplc2xpbnQtZW52IGVzNiovXG5cbnZhciB7IGE6IFt4XX0gPSB7fTsiLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOjExLCJzb3VyY2VUeXBlIjoibW9kdWxlIiwiZWNtYUZlYXR1cmVzIjp7ImpzeCI6dHJ1ZX19LCJydWxlcyI6e30sImVudiI6eyJicm93c2VyIjp0cnVlLCJlczIwMjAiOnRydWUsIm5vZGUiOnRydWUsImVzMjAxNyI6dHJ1ZSwiZXM2Ijp0cnVlfX19)

```js
/*eslint id-length: ["error", { "min": 4 }]*/
/*eslint-env es6*/

var { a: [x]} = {}; // Error: Identifier name `a` ...
```

And there is a PR working on to fix it (#12839).

 > Do we have an issue outlining our plan for this (just in case someone else needs to pick this work up)?

I will add an outlining plan for this on #12832. Thanks 👍